version: '3.8'

services:
  app:
    build:
      context: ./docker/app
    container_name: peligro-symfony-app
    working_dir: /var/www/html
    volumes:
      - ./symfony:/var/www/html
    environment:
      - APP_ENV=local
      - APP_DEBUG=true
      - APP_KEY=base64:K0QkoJdjC83TX5sJ76/O1XLzp5nN+G6XG8uhxdjBzQM=
    ports:
      - "8000:8000"
    networks:
      - custom_network
    

  nginx:
    image: nginx:alpine
    container_name: symfony-nginx
    volumes:
      - ./symfony:/var/www/html
      - ./docker/nginx/default.conf:/etc/nginx/conf.d/default.conf
    ports:
      - "8080:80"
    depends_on:
      - app
    networks:
      - custom_network
  


networks:
  custom_network:
    driver: bridge
    ipam:
      config:
        - subnet: 172.42.12.0/28
          gateway: 172.42.12.1
